examples/example.py

```diff
@@ -1,3 +1,6 @@
+# Python 2/3 compatible codebase
+from __future__ import absolute_import, division, print_function
```

Comment:
Why did you add this?

Reply:
The original last line on examples/example.py is print header.json_string() instead of print (header.json_string()) . I thought it might be intentional not an error or typho to keep the python2  version instead of moving to python3 by just change to print().  `from __future__ import absolute_import, print_function` enable the scripts backward compatibility with python2 while moving to python3 version.

Reply:
NEAT!!